extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -36,7 +39,7 @@
 			<button class="wds-button wds-global-navigation__search-submit">
 				<?= DesignSystemHelper::getSvg(
 					'wds-icons-arrow',
-					'wds-icon wds-icon-small wds-global-navigation__search-icon'
+					'wds-icon wds-icon-small wds-global-navigation__search-submit-icon'
```

Comment:
Sure, but we'd need a message for the alt attribute, which I don't think we have right now. Should that be part of this ticket, or another?